proto/storage/signature_integration.proto

```diff
@@ -19,3 +21,28 @@ message CosignPublicKeyVerification {
   }
   repeated PublicKey public_keys = 3;
 }
+
+// Holds all verification data for verifying certificates attached to cosign signatures.
+// In case both the certificate and chain are given, both will be used to verify the public key
+// as well as the chain.
+// If only the certificate is given, the Fulcio trusted root chain will be assumed and verified against.
+// If only the chain is given, this will be used over the Fulcio trusted root chain for verification.
+// If no certificate or chain is given, the Fulcio trusted root chain will be assumed and verified against.
+message CosignCertificateVerification {
+  // PEM encoded certificate chain to use for verification.
+  string certificate_chains_pem_enc = 1;
```

Comment:
nit: the ordering is inconsistent, in the comment above and in `CosignSignature` the cert comes before the chain. But in this message chain is defined before the cert. Imo it would flow better if this is aligned.